dev-docs/modules/captifyRtdProvider.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: page_v2
+title: Captify RTD Module
+display_name: Captify RTD Module
+description: Captify Real Time Data Module
+page_type: module
+module_type: rtd
+module_code : captifyRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+# Captify RTD Module
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Description
+
+Captify uses publisher first-party on-site search data to power machine learning algorithms to create a suite of
+contextual based targeting solutions that activate in a cookieless environment.
+
+The RTD submodule allows bid requests to be classified by our live-classification service,
+maximising value for publishers by increasing scale for advertisers.
+
+Segments will be attached to bid request objects sent to different SSPs in order to optimize targeting.
+
+Contact prebid@captify.tech for more information.
+## Integration
+
+1) Compile the Captify RTD Module along with your bid adapter and other modules into your Prebid build:  
+
+
+```
+gulp build --modules="rtdModule,captifyRtdProvider,appnexusBidAdapter,..."  
+```
+
+2) Use `setConfig` to instruct Prebid.js to initialize the Captify RTD module, as specified below. 
+
+## Configuration
+
+This module is configured as part of the `realTimeData.dataProviders`  
+
+```javascript
+var TIMEOUT = 1500; // in milliseconds
```

Comment:
1500 ms is way way (way) too long to give as an example auction delay. Please pick something under 100ms for the example. If you choose anything over 100, we will require a comment on the "waitForIt" parameter suggesting that publishers should consider your endpoint's response optional.

Reply:
Hey @bretg ! Thanks for comments!
Currently we got request done inside of this module to our server, my worry was only about users with slow connection, server responding pretty fast (`< 100ms`), this configuration was used for POC testing, it's very likely we will have lower value in prod.

I'm happy to set this to `100ms`

![image](https://user-images.githubusercontent.com/24776415/203754519-c7b5e63e-ea95-4d0f-9eca-249808b244d1.png)